lms/services/canvas_api/_authenticated.py

```diff
@@ -35,28 +37,27 @@ def __init__(  # pylint: disable=too-many-arguments
         self._client_secret = client_secret
         self._redirect_uri = redirect_uri
 
-    def send(self, method, path, schema, params=None):
+    def send(
+        self, method, path, schema, timeout=DEFAULT_TIMEOUT, params=None
+    ):  # pylint:disable=too-many-arguments
         """
         Send a Canvas API request, and retry it if there are OAuth problems.
 
-        :param method: HTTP method to use (e.g. "POST")
-        :param path: The path in the API to make a request to
-        :param schema: Schema to apply to the return values
-        :param params: Any query parameters to add to the request
+        See BasicClient.send() for documentation of parameters, return value
+        and exceptions raised.
```

Comment:
This docstring was mostly a duplication of `BasicClient.send()`'s docstring so I removed it